### ISSUE TITLE
Broken dependency rsc.io/quote

### ISSUE THREAD

**Issue by Warfields** [golang/mock#464]
**Actual behavior** A clear and concise description of what the bug is.

```
$ export GOPROXY=direct
$ go get github.com/golang/mock/
go: github.com/golang/mock@v1.4.3 requires
	rsc.io/quote/v3@v3.1.0: unrecognized import path "rsc.io/quote/v3": reading https://rsc.io/quote/v3?go-get=1: 503 Service Unavailable


```
**Expected behavior** A clear and concise description of what you expected to
happen.

Go get to work properly

**To Reproduce** Steps to reproduce the behavior

```
$ export GOPROXY=direct
$ go get github.com/golang/mock/
```

**Additional Information**

-   gomock mode (reflect or source):
-   gomock version or git ref: 
-   golang version: 1.14

**Triage Notes for the Maintainers**

**Comment by Warfields** [comment]
Someone posted another bug report at the same time

**Comment by Warfields** [comment]
Merged into #463
